### PR TITLE
Document how to run the tests locally using docker-compose

### PR DESCRIPTION
**Pull request by badboy**

### CODE REVIEW

File: application/README.md [hunk 1]
```diff
@@ -60,6 +60,19 @@ install_requires=["click>=7.0", "glean-sdk==31.2.1", "wrapt", "typing_extensions
...
+## Run end-to-end tests locally
+
+You can run the `fake-data-platform` and the burnham missions locally using `docker-compose`.
+Run the following command from the top-level directory:
+
+```
+docker-compose up --build platform cli1
+```
+
+This will launch the platform and run 3 additional containers running `burnham`.
```
Comment (hackebrot) [inline application/README.md:72]: Docker will only run 3 burnham containers with the default config in `docker-compose.yml`.

**Review by hackebrot — CHANGES_REQUESTED** [review]

File: application/README.md (hunk 1 above)
Comment (badboy) [reply application/README.md:72]: and the platform container, no?

Comment (hackebrot) [reply application/README.md:72]: Correct! I was merely pointing out that the above `docker-compose` command will run only a single cli client.

Comment (badboy) [reply application/README.md:72]: Woops, because I copied my modified one, this should of course leave out the names.

**Review by hackebrot — APPROVED** [review]
LGTM! 📝

**Pull request merged**